README.md

```
@@ -1,69 +1,49 @@
-# 🌱 DiverCompass
+#🌊 DivyDot
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading
null

(MD018, no-missing-space-atx)